### DISCUSSION TITLE
unable to install linkerd on AKS

### DISCUSSION THREAD

**Discussion by AnnapoornaKaradi** [linkerd/linkerd2#13224] [Help]
I m trying to install linkerd stable-2.14.10 on an AKS cluster through helm charts, when i deploy the pods are going in crashloopbackoff For destination -- [   146.031635s] ERROR ThreadId(02) identity: linkerd_proxy_identity_client::certify: Failed to obtain identity error=status: Unknown, message: "controller linkerd-identity-headless.linker  d.svc.cluster.local:8080: service in fail-fast", details: [], metadata: MetadataMap { headers: {} } error.sources=[controller linkerd-identity-headless.linkerd.svc.cluster.local:8080: serv  ice in fail-fast, service in fail-fast]     for identity -- time="2024-10-24T10:09:27Z" level=info msg="running version stable-2.14.10"  time="2024-10-24T10:09:27Z" level=info msg="starting admin server on :9990"  time="2024-10-24T10:09:27Z" level=info msg="Using k8s client with QPS=100.00 Burst=200"  time="2024-10-24T10:09:27Z" level=fatal msg="Failed to initialize identity service: Post \https://10.0.0.1:443/apis/authorization.k8s.io/v1/selfsubjectaccessreviews\: read tcp 10.151.34.  18:37438->10.0.0.1:443: read: connection reset by peer"        For proxy-injector -- time="2024-10-24T10:09:24Z" level=info msg="running version stable-2.14.10"  time="2024-10-24T10:09:24Z" level=info msg="starting admin server on :9995"  time="2024-10-24T10:09:24Z" level=info msg="waiting for caches to sync"  time="2024-10-24T10:09:24Z" level=info msg="listening at :8443"  time="2024-10-24T10:09:36Z" level=info msg="Trace[1438484023]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.477) (tot  al time: 12236ms):\nTrace[1438484023]: ---\"Objects listed\" error:<nil> 12235ms (10:09:36.713)\nTrace[1438484023]: [12.236303098s] [12.236303098s] END"  time="2024-10-24T10:09:36Z" level=info msg="Trace[1167630554]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.478) (tot  al time: 12402ms):\nTrace[1167630554]: ---\"Objects listed\" error:<nil> 12402ms (10:09:36.881)\nTrace[1167630554]: [12.402894161s] [12.402894161s] END"  time="2024-10-24T10:09:37Z" level=info msg="Trace[1720575672]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.477) (tot  al time: 12611ms):\nTrace[1720575672]: ---\"Objects listed\" error:<nil> 12611ms (10:09:37.089)\nTrace[1720575672]: [12.61139541s] [12.61139541s] END"  time="2024-10-24T10:09:37Z" level=info msg="Trace[2113538521]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.478) (tot  al time: 12802ms):\nTrace[2113538521]: ---\"Objects listed\" error:<nil> 12802ms (10:09:37.280)\nTrace[2113538521]: [12.802816648s] [12.802816648s] END"  time="2024-10-24T10:09:37Z" level=info msg="Trace[1915986715]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.477) (tot  al time: 13002ms):\nTrace[1915986715]: ---\"Objects listed\" error:<nil> 13002ms (10:09:37.479)\nTrace[1915986715]: [13.002083269s] [13.002083269s] END"  time="2024-10-24T10:09:37Z" level=info msg="Trace[2055050403]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.477) (tot  al time: 13202ms):\nTrace[2055050403]: ---\"Objects listed\" error:<nil> 13202ms (10:09:37.679)\nTrace[2055050403]: [13.202097806s] [13.202097806s] END"  time="2024-10-24T10:09:37Z" level=info msg="Trace[1529451565]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.477) (tot  al time: 13402ms):\nTrace[1529451565]: ---\"Objects listed\" error:<nil> 13402ms (10:09:37.880)\nTrace[1529451565]: [13.402621819s] [13.402621819s] END"  time="2024-10-24T10:09:38Z" level=info msg="Trace[2032762245]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.477) (tot  al time: 13604ms):\nTrace[2032762245]: ---\"Objects listed\" error:<nil> 13604ms (10:09:38.082)\nTrace[2032762245]: [13.604168047s] [13.604168047s] END"  time="2024-10-24T10:09:38Z" level=info msg="Trace[856760753]: \"Reflector ListAndWatch\" name:pkg/mod/k8s.io/client-go@v0.29.0/tools/cache/reflector.go:229 (24-Oct-2024 10:09:24.478) (tota  l time: 13818ms):\nTrace[856760753]: ---\"Objects listed\" error:<nil> 13817ms (10:09:38.295)\nTrace[856760753]: [13.818053947s] [13.818053947s] END"  time="2024-10-24T10:09:38Z" level=info msg="caches synced"

**Comment by kflynn** [comment]
Hey @AnnapoornaKaradi, it's very hard to read that log, but it kind of looks like the API server isn't answering when the identity controller tries to talk to it? Asking the obvious question, is the cluster healthy? Which chart or charts did you try to install?
